source/auth/auth.md

```diff
@@ -959,6 +959,11 @@ Examples are provided below.
 
         Drivers MUST allow the user to specify an AWS session token for authentication with temporary credentials.
 
+    - AWS_CREDENTIAL_PROVIDER
+
+        Drivers MAY allow the user to specify a custom credential provider object or function. See
+        [Custom Credential Providers](https://github.com/mongodb/specifications/blob/master/source/auth/auth.md#custom-credential-providers)
```

Comment:
Can we make this a relative anchor tag link similar to `[MONGODB-OIDC](#mongodb-oidc)` in the Reauthentication section?